core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.component.ts

```diff
@@ -155,11 +155,16 @@ export class CreateNewSubtopicModalComponent
 
   isUrlFragmentValid(): boolean {
```

Comment:
No, I only changed this because there is a function ```(change)="checkSubtopicExistence()"```.

Reply:
@Somnath-Mishra OK, but then, should you update so that the logic is the same throughout all the files? If not, then what makes this case special?

Also please see the [general question](https://github.com/oppia/oppia/pull/21184#pullrequestreview-2469381676) I left, above.

Reply:
@seanlip I will create a shared ```url-fragment-editor``` component that can be used in several places.